packages/beachball/src/changelog/renderPackageChangelog.ts

```diff
@@ -1,10 +1,30 @@
-import { PackageChangelog, ChangelogEntry } from '../types/ChangeLog';
+import { PackageChangelog, ChangelogEntry, ChangelogJsonEntry } from '../types/ChangeLog';
 import _ from 'lodash';
+import { GitHubInfo } from '../types/BeachballOptions';
+
+export interface PackageChangelogRenderOptions {
+  changelog: PackageChangelog;
+  isGroupedChangelog: boolean;
+  previousChangelogEntry: ChangelogJsonEntry | undefined;
+  github?: GitHubInfo;
+}
+
+export function renderPackageChangelog(options: PackageChangelogRenderOptions): string {
+  const { changelog, isGroupedChangelog, previousChangelogEntry, github } = options;
+  const header =
+    github && changelog.tag
+      ? `[${changelog.version}](https://github.com/${github.owner}/${github.repo}/tree/${changelog.tag})`
+      : changelog.version;
+
+  let subHeader = changelog.date.toUTCString();
+  subHeader +=
+    github && previousChangelogEntry?.tag
+      ? `\n[Compare changes](https://github.com/${github.owner}/${github.repo}/compare/${previousChangelogEntry?.tag}..${changelog.tag})`
```

Comment:
We should definitely not make this so github specific. This is where I wished we had a custom renderer function that folks can use. One way to work around having a custom function would be to allow for a template string or a function...

Reply:
If we do have a custom render function, maybe we could provide an OOB one (or an option) which works with github

Reply:
This will also be a design issue relevant to the feature I'm working on, retrieving and adding PR links to changelogs.

Reply:
@kenotron @ecraig12345 
I went with supporting custom render function: https://github.com/microsoft/beachball/pull/289/files

let me know what you think